src/auslib/web/public/json.py

```diff
@@ -9,15 +9,22 @@
 from auslib.web.public.helpers import AUS, with_transaction
 
 
+def get_aus_metadata_headers(eval_metadata):
+    header_name_metadata_map = {"rule": "Rule-ID", "rule_data_version": "Rule-Data-Version"}
+    headers = {header_name_metadata_map.get(name, name): value for name, value in eval_metadata.items()}
+    return headers
+
+
```

Comment:
This is the same as `client.py` `get_aus_metadata_headers` function. Let me know if is better move to a commom place.

Reply:
https://github.com/mozilla-releng/balrog/blob/main/src/auslib/web/public/helpers.py is probably a good place for it.